tests/collections/asr/test_asr_modules.py

```diff
@@ -38,13 +52,178 @@ def test_AudioToMelSpectrogramPreprocessor1(self):
             length = torch.randint(low=161, high=500, size=[4])
             res1, length1 = instance1(input_signal=input_signal, length=length)
             res2, length2 = instance2(input_signal=input_signal, length=length)
-            for len1, len2 in zip(length1, length2):
-                assert len1 == len2
-            assert res1.shape == res2.shape
-            diff = torch.mean(torch.abs(res1 - res2))
-            assert diff <= 1e-3
-            diff = torch.max(torch.abs(res1 - res2))
-            assert diff <= 1e-2
+            self.compare_features(res1, length1, res2, length2)
+
+    @pytest.mark.unit
+    def test_AudioToMelSpectrogramPreprocessor_DALI_spectrogram(self):
```

Comment:
Could you mark the test as optional (in case Dali is not installed by user) - similar to here https://github.com/NVIDIA/NeMo/blob/main/tests/collections/asr/test_asr_rnnt_encdec_model.py#L95-L97

Reply:
Done.

Reply:
Thanks